### PR TITLE
 Refs #26346 - tasks table uses router history

### PR DESCRIPTION
**Pull request by MariaAga**
Instead of using `window.history`

### CODE REVIEW

**Comment by adamruzicka** [comment]
Needs a rebase

**Comment by sharvit** [comment]
Looking at the commits, looks like need a rebase?

File: webpack/ForemanTasks/Components/TasksTable/TasksTable.js [hunk 1]
```diff
@@ -50,11 +40,31 @@ const TasksTable = ({
...
+      },
+      history
+    );
+  };
+
+  const setSortHistory = (by, order) => {
+    updateURlQuery({ sort_by: by, sort_order: order }, history);
+  };
+
+  const uriQuery = getURIQuery(window.location.href);
```
Comment (sharvit) [inline webpack/ForemanTasks/Components/TasksTable/TasksTable.js]: Can't it use the `history` object as well?

Comment (MariaAga) [reply webpack/ForemanTasks/Components/TasksTable/TasksTable.js]: You're right! changed here and in other places that I missed as well

**Comment by adamruzicka** [comment]
Thank you @MariaAga for the patch, thank you @sharvit and @waldenraines for reviews.

**Pull request merged**